src/read_utils.jl

```diff
@@ -57,6 +57,21 @@ function analyze_symmetry_data(
     return band_summaries
 end
 
+function fix_gamma_irrep(symeigsd::Dict{String, Vector{Vector{ComplexF64}}}, lgd::Dict{String, LittleGroup{D}}, mode::AbstractString) where D
```

Comment:
Maybe just take the Γ little group instead of an entire dictionary at Γ? I.e. change `lgd::Dict{String, LittleGroup{D}}` to `lg::LittleGroup{D}`.

Then you can always have another method of `fix_gamma_irrep` that takes `lgd` as a convenience accessor.

Also, let's give `mode` a default (of `nothing`) and rename it to `polarization`.